### PR TITLE
Some launch files and memory leak fixes

### LINKED ISSUE #72: Make queue_size=1 everywhere
Having `queue_size=0` make ros use an unlimited sized queue, which just explodes your computer's RAM 🙃. If you think you have a memory leak because you're going OOM, maybe you just have this issue.

@kpberry is curently fixing this on the `launch` branch. I'm mainly making this issue for future reference.

### PR DESCRIPTION
**Pull request by kpberry**
Fixes #72 #62 #61
@kpberry did I miss any issues?

### CODE REVIEW

File: buzzmobile/process/frame_merger/frame_merger.py [hunk 1]
```diff
@@ -15,7 +14,7 @@ def __init__(self):
 
 # Global Variables
 bridge = CvBridge()
-world_pub = rospy.Publisher('world_model', Image, queue_size=0)
+world_pub = rospy.Publisher('world_model', Image, queue_size=1)
```
Comment (irapha) [inline buzzmobile/process/frame_merger/frame_merger.py:17]: For future reference, having `queue_size=0` make ros use an unlimited sized queue, which just explodes your computer's RAM :)

**Review by irapha — APPROVED** [review]

**Comment by joshuamorton** [comment]
drat I missed this, does this include all python dependencies, I'm not seeing the cv2, matplotlib, or tensorflow bindings, or are those done elsewhere, or requests, which is missing.

**Comment by kpberry** [comment]
It doesn't, but I should be able to add them in a few minutes.

**Comment by joshuamorton** [comment]
Its also similarly likely that we'll almost immediately need to pull everything out into a build.sh file to do the tf, matplot, and cv2 stuff, but that's fine, better to have it all in one place.

**Comment by irapha** [comment]
Cv3 has to be installed in a special way. The details as to why are in an
issue you can dig up if you want.

Neither matplotlib not tensorflow are currently dependencies in Buzzmobile.

**Comment by irapha** [comment]
Idk why we would depend on matplotlib tbh. But yes there's an open issue
for standardizing installation.

**Pull request merged**